include/lsst/daf/base/DateTime.h

```diff
@@ -63,15 +63,15 @@ class DateTime {
     explicit DateTime(double date, DateSystem system = MJD, Timescale scale = TAI);
     DateTime(int year, int month, int day, int hr, int min, int sec,
              Timescale scale = TAI);
-    explicit DateTime(std::string const& iso8601);
+    explicit DateTime(std::string const& iso8601, Timescale scale);
```

Comment:
Yes. It's a bit loose. http://docs.astropy.org/en/v1.2.1/time/index.html has some examples and I remember getting very confused when plugging `astropy.time` in as a backend for `DateTime`.